src/components/inputs/date/ohri-date.component.tsx

```diff
@@ -40,7 +40,12 @@ const OHRIDate: React.FC<OHRIFormFieldProps> = ({ question, onChange, handler })
     setFieldValue(question.id, refinedDate);
     onChange(question.id, refinedDate, setErrors);
     onTimeChange(false, true);
-    question.value = handler.handleFieldSubmission(question, refinedDate, encounterContext);
+    moment(refinedDate.value).format('YYYY-MM-DD HH:mm');
```

Comment:
@ODORA0 does this line have any side-effect?

Reply:
@samuelmale Not sure what you asking